onedal/utils/validation.py

```diff
@@ -438,3 +443,31 @@ def _is_csr(x):
     return isinstance(x, sp.csr_matrix) or (
         hasattr(sp, "csr_array") and isinstance(x, sp.csr_array)
     )
+
+
+def _assert_all_finite(X, allow_nan=False, input_name=""):
```

Comment:
Is a version check required here that sets `_assert_all_finite()` to the `daal4py` version for older oneDAL releases?

Reply:
Changes made for circumstance (won't come up in any of the CIs though)